scripts/qa_crowbarsetup.sh

```diff
@@ -3405,11 +3405,15 @@ function onadmin_proposal
     fi
     local proposal
     for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+        ceph monasca glance cinder neutron ; do
         deploy_single_proposal $proposal
     done
+    for proposal in nova `horizon_barclamp` ceilometer heat manila trove \
+        barbican magnum sahara murano aodh tempest; do
+        deploy_single_proposal $proposal
+        get_novacontroller
```

Comment:
I think that we call this for the side effect of setting `novacontroller` global variable, so we can put this out of the loop